src/beheer-footer/beheer-footer.scss

```diff
@@ -1,12 +1,11 @@
-// this var is needed to build, but will be overridden by the app that implements the component
-$static-path: 'static';
+$pyoes-img-path: '' !default;
```

Comment:
gaat deze ooit kunnen overschreven worden als beheer-header.css in de html template als gerequired wordt?

Reply:
enkel als je deze scss file nog eens importeert.

Reply:
bij nader inzien is het dan simpeler wss om deze var weg te laten en gewoon in een custom.scss de url aan te passen indien nodig

Reply:
> scss



> gaat deze ooit kunnen overschreven worden als beheer-header.css in de html template als gerequired wordt?

https://github.com/OnroerendErfgoed/inventaris/pull/5175/files#diff-622a99dbd389325a24320111cd1b7a46733b38fbf3c93dcc220b29c5f733cf3eR10

Daar doen we dat, als dat is wat ge bedoelt?

Reply:
ja check. ma waarom dan nog in den template zelf diene import?

Reply:
deze import he en in beheer-footer? https://github.com/OnroerendErfgoed/oerelia/blob/51f1921a2ac6cf5ca0423ea43036b07c84b2663a/src/beheer-header/beheer-header.html#L2

sure die kan dan weg als we toch enkel met de var willen werken